README.adoc

```diff
@@ -3216,6 +3216,42 @@ def some_method(&)
 end
 ----
 
+=== Super Forwarding
+
+Use `super` without arguments when the enclosing methods arguments are the same.
+
+When super is called without arguments and parentheses Ruby will automatically forward the arguments taken by the method.
+
+[source,ruby]
+----
+# bad
+def some_method(*args, **kwargs)
+  super(*args, **kwargs)
+end
+
+# good - implicitly passing all arguments
+def method(*args, **kwargs)
+  super
+end
+
+# good - forwarding a subset of the arguments
+def method(*args, **kwargs)
+  super(*args)
+end
+
+# good - calling super with different arguments
+def method(*args, **kwargs)
+  super("foo", *args, **kwards)
+end
+
+# good - forwarding no arguments
+def method(*args, **kwargs)
+  super()
+end
+----
+
+IMPORTANT: Blocks are always passed to the parent method. `&nil` can be passed to `super` if the block should remain local.
```

Comment:
What if there are other parameters? What is the recommended way to pass all arguments to super, but not the block? The … won’t work, will it?

Reply:
If you don't want to pass the block you write `&nil`, so in these examples it would be `super(*args, **kwargs, &nil)`. `super(..., &nil)` on the other hand is a syntax error so it doesn't look like you are able to suppress the block with that notation.

Should I make a explicit example instead of just mentioning it here?